README.md

```diff
@@ -105,6 +106,20 @@ aggregate?: (expression: Alg.AggregateExpression) => Promise<RDF.Term>;
 
 You can probably ignore this.
 
+### Extension functions
+
+Extension functions can be added by providing the `extensionFunctionCreator` in the config.
+Example
+```ts
+config.extensionFunctionCreator = (functionName: RDF.NamedNode) => {
+   if (functionNamedNode.value === 'https://example.org/functions#equal') {
+      return async (args: RDF.Term[]) => {
+         return literal(String(args[0].equals(args[1])), 'http://www.w3.org/2001/XMLSchema#boolean');       
+      }
+   }
+}
+```
+
 ### Custom functions
 
 **NOTE: Not yet implemented. On the roadmap.**
```

Comment:
This can now go? :-)

Reply:
If you `custom function`, I don't think so, as Wout said, we should make a distinction between custom function and extension function. What we did is the extension function. I think we should leave it there because we are still planning on implementing custom functions right?

Reply:
Well, at the moment, most readers of the README will not know the difference between the two, and this will cause confusion.

I think the "Custom functions" section was meant for the extension functions we have here.
The custom functions you refer to would be more like function _overrides_, which can indeed be added in the future, but doesn't seem that critical do add in the README as something that is coming soon.

Reply:
You'e right, I'll remove it from the README.